install_config/aggregate_logging.adoc

```diff
@@ -666,13 +672,19 @@ operations cluster if present, just with the names changed to include `-ops`.
 [[aggregated-elasticsearch]]
 === Elasticsearch
 
-A highly-available environment requires at least three replicas of
-Elasticsearch; each on a different host. Elasticsearch replicas require their
-own storage, but an {product-title} deployment configuration shares storage
-volumes between all its pods. So, when scaled up, the EFK deployer ensures each
-replica of Elasticsearch has its own deployment configuration.
+link:https://www.elastic.co/products/elasticsearch[Elasticsearch (ES)] is an object store where all logs are stored.
+ 
+Elasticsearch organizes the log data into datastores, each called an _index_. Elasticsearch subdivides each index 
+into multiple pieces called _shards_. Elasticsearch spreads the shards across a set of Elasticsearch nodes in your cluster. 
+You can configure Elasticsearch to make copies of the shards, called _replicas_. Elasticsearch also spreads replicas across 
+the Elactisearch nodes.  The combination of shards and replicas is intended to 
+provide redundancy and resilience to failure. For example, if you configre three shards for the index with one replica, 
+Elasticsearch generates a total of six shards for that index: three primary shards and three replicas as a backup.
 
-It is possible to scale your cluster up after creation by modifying the
+The {product-title} installer ensures each Elasticsearch node is deployed using a unique deployment configuration that includes its own storage volume. 
+You must xref:scaling-elasticsearch[create an additional deployment configuration] for each Elasticsearch node you add to the logging system. 
```

Comment:
You can also mention the installer will do this if you bump openshift_logging_es_cluster_size or back ref it some how